### ISSUE TITLE
Getting multiple errors in the logs.

### ISSUE THREAD

**Issue by adame21** [gotenberg/gotenberg#288]
Hello, 
I've searched before posting and couldn't find anything.

## Expected Behavior
Office and HTML to PDF conversions

## Current Behavior
Many failures and suspicious logs.

## Possible Solution
Have looked around google and some of these seem to be related to chromium

## Steps to Reproduce (for bugs)
I'm currently running with these resources:
````yaml
    component_replicas_min: 2
    component_replicas_max: 4
    component_cpu_limits: 400m
    component_cpu_requests: 200m
    component_mem_limits: 2048Mi
    component_mem_requests: 512Mi
````
and these env vars
````yaml
  MAXIMUM_WAIT_TIMEOUT: "360"
  DEFAULT_WAIT_TIMEOUT: "360" 
  LOG_LEVEL: "DEBUG"
````

## Logs (LOG_LEVEL="DEBUG")
I looked around the logs and found these:
1.
````json
{"bytes_in":"0","bytes_out":"36","host":"commons-gotenberg:3000","latency":"360006121982","latency_human":"6m0.006121982s","level":"error","method":"POST","msg":"request failed","op":"context.Context.LogRequestResult","path":"/convert/html","referer":"","remote_ip":"10.122.228.132","status":504,"time":"2021-03-08T11:47:34Z","trace":"m23s6pvHHgEY1VBpeOovLRBSQ8q8sZtH","uri":"/convert/html","user_agent":""}
````
2.
````json
{"level":"debug","msg":"[0210/190311.095311:ERROR:zygote_host_impl_linux.cc(262)] Failed to adjust OOM score of renderer with pid 10381: Permission denied (13)","op":"stderr.google-chrome-stable.--no-sandbox.--headless.--disable-dev-shm-usage.--font-render-hinting=none.--remote-debugging-port=9222.--disable-gpu.--disable-translate.--disable-extensions.--disable-background-networking.--safebrowsing-disable-auto-update.--disable-sync.--disable-default-apps.--hide-scrollbars.--metrics-recording-only.--mute-audio.--no-first-run","time":"2021-02-10T19:03:11Z","trace":"system"}
````
3. Just some more possibly important logs
````json
"level":"debug","msg":"resource directory 'QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV' created","op":"resource.New","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"skipping 'Content-Type' as it is not a custom HTTP header...","op":"resource.Resource.WithCustomHTTPHeader","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"skipping 'Connection' as it is not a custom HTTP header...","op":"resource.Resource.WithCustomHTTPHeader","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'resultFilename' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'waitTimeout' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'webhookURL' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'webhookURLTimeout' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'remoteURL' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'waitDelay' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'paperWidth' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'paperHeight' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'marginTop' with value '0' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'marginBottom' with value '0' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'marginLeft' with value '0.2' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'marginRight' with value '0.2' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'landscape' with value 'false' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'pageRanges' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'googleChromeRpccBufferSize' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"added 'scale' with value '' to resource args","op":"resource.Resource.WithArg","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"resource file 'index.html' created","op":"resource.Resource.WithFile","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"handling HTML request...","op":"xhttp.htmlHandler","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"no 'webhookURL' found, converting synchronously","op":"xhttp.convert","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"options: {WaitTimeout:360 WaitDelay:0 HeaderHTML:\u003chtml\u003e\u003chead\u003e\u003c/head\u003e\u003cbody\u003e\u003c/body\u003e\u003c/html\u003e FooterHTML:\u003chtml\u003e\u003chead\u003e\u003c/head\u003e\u003cbody\u003e\u003c/body\u003e\u003c/html\u003e PaperWidth:8.27 PaperHeight:11.7 MarginTop:0 MarginBottom:0 MarginLeft:0.2 MarginRight:0.2 Landscape:false PageRanges: RpccBufferSize:1048576 CustomHTTPHeaders:map[] Scale:1}","op":"printer.logOptions","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"creating context with '360.00s' of timeout...","op":"xcontext.WithTimeout","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"skipping lock acquisition...","op":"printer.chromePrinter.Print","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"[0207/113601.850139:ERROR:zygote_host_impl_linux.cc(262)] Failed to adjust OOM score of renderer with pid 139: Permission denied (13)","op":"stderr.google-chrome-stable.--no-sandbox.--headless.--disable-dev-shm-usage.--font-render-hinting=none.--remote-debugging-port=9222.--disable-gpu.--disable-translate.--disable-extensions.--disable-background-networking.--safebrowsing-disable-auto-update.--disable-sync.--disable-default-apps.--hide-scrollbars.--metrics-recording-only.--mute-audio.--no-first-run","time":"2021-02-07T11:36:01Z","trace":"system"}
{"level":"debug","msg":"skipping custom HTTP headers as none have been provided...","op":"printer.chromePrinter.setCustomHTTPHeaders","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'init' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:01Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'firstPaint' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:02Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'firstContentfulPaint' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:02Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'firstMeaningfulPaintCandidate' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:02Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'loadingFinished' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:03Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'firstImagePaint' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:03Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'DOMContentLoaded' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:04Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'domContentEventFired' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:04Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'load' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:04Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'loadEventFired' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:04Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'networkAlmostIdle' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:05Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'firstMeaningfulPaint' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:05Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"event 'networkIdle' received","op":"printer.chromePrinter.listenEvents","time":"2021-02-07T11:36:05Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"no wait delay to apply, moving on...","op":"printer.chromePrinter.Print","time":"2021-02-07T11:36:05Z","trace":"QIVOIUJ4gDOLk6LQmVZXYJqm9Bes5ORV"}
{"level":"debug","msg":"handling ping request...","op":"xhttp.pingHandler","time":"2021-02-07T11:36:05Z","trace":"TSpQy0l9s8PxO2MX50YmVEMKgyBT5cLJ"}
````

**Comment by gulien** [comment]
@adame21 If the issue still occurs with v7, please reopen this issue!

**Comment by Cluster2a** [comment]
@gulien, I am still getting the error `Failed to adjust OOM score of renderer with pid 139: Permission denied` in 8.13.0.

**Comment by peterfarrell** [comment]
Ditto, same issue on Gotenberg 8.14.1 deployed on OpenShift. Below is an error of a failed called to merge multiple PDFs.

```shell
{"level":"debug","ts":1736784989.9607677,"logger":"api.formslibreofficeconvert.usrbinunoconverter.stderr","msg":"  from distutils.version import LooseVersion","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
{"level":"debug","ts":1736784990.1542842,"logger":"api.formslibreofficeconvert.usrbinunoconverter.stderr","msg":"Verbosity set to level 3","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
{"level":"debug","ts":1736784990.155243,"logger":"api.formslibreofficeconvert.usrbinunoconverter.stderr","msg":"DEBUG: Connection type: socket,host=127.0.0.1,port=46495,tcpNoDelay=1;urp;StarOffice.ComponentContext","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
{"level":"debug","ts":1736784990.1692233,"logger":"api.formslibreofficeconvert.usrbinunoconverter.stderr","msg":"Input file: /tmp/ee5b31d7-231d-4ad8-b338-0e0961484fc5/1241e6d9-cb42-4359-81ea-05b0ca62e83d/pinia-cheat-sheet.pdf","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
{"level":"debug","ts":1736785025.9523103,"logger":"chromium.browser","msg":"[18:52:0113/161705.952168:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 272: Permission denied (13)\n"}
{"level":"debug","ts":1736785028.157807,"logger":"chromium.browser","msg":"[18:52:0113/161708.157681:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 278: Permission denied (13)\n"}
{"level":"debug","ts":1736785029.2575743,"logger":"chromium.browser","msg":"[18:52:0113/161709.257439:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 284: Permission denied (13)\n"}
{"level":"debug","ts":1736785030.1567538,"logger":"chromium.browser","msg":"[18:52:0113/161710.156635:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 291: Permission denied (13)\n"}
{"level":"debug","ts":1736785031.2554379,"logger":"chromium.browser","msg":"[18:52:0113/161711.255297:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 297: Permission denied (13)\n"}
{"level":"debug","ts":1736785032.6561651,"logger":"chromium.browser","msg":"[18:52:0113/161712.656015:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 303: Permission denied (13)\n"}
{"level":"debug","ts":1736785034.3565319,"logger":"chromium.browser","msg":"[18:52:0113/161714.356354:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 309: Permission denied (13)\n"}
{"level":"debug","ts":1736785036.5518093,"logger":"chromium.browser","msg":"[18:52:0113/161716.551577:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 315: Permission denied (13)\n"}
{"level":"debug","ts":1736785039.2588673,"logger":"chromium.browser","msg":"[18:52:0113/161719.258739:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 321: Permission denied (13)\n"}
{"level":"debug","ts":1736785042.4554555,"logger":"chromium.browser","msg":"[18:52:0113/161722.455314:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 327: Permission denied (13)\n"}
{"level":"debug","ts":1736785046.2517786,"logger":"chromium.browser","msg":"[18:52:0113/161726.251614:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 333: Permission denied (13)\n"}
{"level":"debug","ts":1736785050.2567582,"logger":"chromium.browser","msg":"[18:52:0113/161730.256587:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 339: Permission denied (13)\n"}
{"level":"debug","ts":1736785055.2548332,"logger":"chromium.browser","msg":"[18:52:0113/161735.254656:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 345: Permission denied (13)\n"}
{"level":"debug","ts":1736785061.0547931,"logger":"chromium.browser","msg":"[18:52:0113/161741.054639:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 351: Permission denied (13)\n"}
{"level":"debug","ts":1736785066.5552118,"logger":"chromium.browser","msg":"[18:52:0113/161746.555097:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 357: Permission denied (13)\n"}
{"level":"debug","ts":1736785072.651486,"logger":"chromium.browser","msg":"[18:52:0113/161752.651292:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 363: Permission denied (13)\n"}
{"level":"debug","ts":1736785073.957411,"logger":"chromium.browser","msg":"[18:52:0113/161753.957285:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 369: Permission denied (13)\n"}
{"level":"debug","ts":1736785084.2575498,"logger":"chromium.browser","msg":"[18:52:0113/161804.257423:ERROR:zygote_host_impl_linux.cc(279)] Failed to adjust OOM score of renderer with pid 375: Permission denied (13)\n"}
{"level":"debug","ts":1736785085.1644418,"logger":"chromium.browser","msg":"Warning: vkCreateInstance: Found no drivers!\n"}
{"level":"debug","ts":1736785085.1648414,"logger":"chromium.browser","msg":"Warning: vkCreateInstance failed with VK_ERROR_INCOMPATIBLE_DRIVER\n    at CheckVkSuccessImpl (../../third_party/dawn/src/dawn/native/vulkan/VulkanError.cpp:106)\n\n"}
{"level":"debug","ts":1736785085.4583333,"logger":"chromium.browser","msg":"Warning: vkCreateInstance: Found no drivers!\n"}
{"level":"debug","ts":1736785085.4583716,"logger":"chromium.browser","msg":"Warning: vkCreateInstance failed with VK_ERROR_INCOMPATIBLE_DRIVER\n    at CheckVkSuccessImpl (../../third_party/dawn/src/dawn/native/vulkan/VulkanError.cpp:106)\n\n"}
{"level":"debug","ts":1736785091.0833077,"logger":"api.formslibreofficeconvert.usrbinunoconverter.stderr","msg":"unoconv: RuntimeException during import phase:","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
{"level":"debug","ts":1736785091.0833662,"logger":"api.formslibreofficeconvert.usrbinunoconverter.stderr","msg":"Office probably died. Binary URP bridge disposed during call","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
{"level":"debug","ts":1736785091.084398,"logger":"api.formslibreofficeconvert.usrbinunoconverter.stdout","msg":"Using office base path: /usr/lib/libreoffice","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
{"level":"debug","ts":1736785091.0844166,"logger":"api.formslibreofficeconvert.usrbinunoconverter.stdout","msg":"Using office binary path: /usr/lib/libreoffice/program","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
{"level":"debug","ts":1736785091.1608295,"logger":"api.formslibreofficeconvert.usrbinunoconverter","msg":"unix process already killed","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
{"level":"debug","ts":1736785091.1608799,"logger":"api.formslibreofficeconvert","msg":"process lock released","trace":"29ff3bf4-f9ba-400c-a79a-f9ce30624de2"}
```
